Tests/Functional/Domain/Repository/TaskExecutionRepositoryTest.php

```diff
@@ -40,13 +40,11 @@ public function findNext(): void
             [
                 'taskOne' => [
                     'handlerClass' => TestHandler::class,
-                    'cronExpression' => '5 * * * *',
-
+                    'cronExpression' => '* * * * *',
                 ],
                 'taskTwo' => [
                     'handlerClass' => TestHandler::class,
-                    'cronExpression' => '* * * * *',
-
+                    'cronExpression' => '5 * * * *',
```

Comment:
I'm not perfectly sure if that "fix" is correct (I just swapped the order of the cron expressions basically) and I'm not sure if that test is bound to fail whenever it is executed at a time before 5 minutes past a full hour..